Sources/EliminationMenu.swift

```diff
@@ -98,23 +98,23 @@ public class EliminationMenu: UIView {
         
         if menu.isLeftAligned {
             view.addConstraint(NSLayoutConstraint(
-                item: menu, attribute: .Left, relatedBy: .Equal, toItem: view, attribute: .Left, multiplier: 1, constant: margin.x)
+                item: view, attribute: .LeftMargin, relatedBy: .Equal, toItem: menu, attribute: .Left, multiplier: 1, constant: margin.x)
```

Comment:
Changing of `view` and `menu` would reintroduce the bug i fixed in [this Commit](https://github.com/r-dent/EliminationMenu/commit/11973b8bbd3dbda6f76d5c2268c7f3bcaac6d9bb), i think. (the menu was not inset but "outset" of the containing view.)